src/components/layout/about-section/discover-button.tsx

```
@@ -0,0 +1,17 @@
+import { Button } from "@components/ui/button";
+import { cn } from "@lib/utils/cn.utils";
+import { ArrowRight } from "lucide-react";
```

ESLint

[error] 'ArrowRight' is defined but never used.

(@typescript-eslint/no-unused-vars)